File: Lista 1/Battleship - Batalha Naval/python/Board.py
class Board:
    def __init__(self):
        self.board = [['-' for _ in range(5)] for _ in range(5)]

    def buildBoards(self, user):
        for i in range(5):
            for j in range(5):
                user.board[i][j] = '-'  # preenche todos os espaços com valores em branco

        for i in range(5):
            coordinates = user.coordinates[i]
            row = coordinates[0]  # armazena linha
            column = coordinates[1]  # armazena coluna
            user.board[row][column] = '@'  # encontra coordenada escolhida e coloca um navio nela

        for i in range(5):
            for j in range(5):
                print(user.board[i][j], end=' ')  # imprime tabuleiro inicial
            print()

File: Lista 1/Battleship - Batalha Naval/java/Board.java
public class Board {
    protected char[][] board = new char[5][5];

    public void buildBoards(ReceiveCoordinates user){
        for(int i = 0; i < 5; i++){
            for(int j = 0; j < 5; j++){
                user.board[i][j] = '-';//preenche todos os espaços com valores em branco
            }
        }

        for(int i = 0; i < 5; i++){
            int[] coordinates = user.coordinates[i];
            int row = coordinates[0];//armazena linha
            int column = coordinates[1];//armazena coluna
            user.board[row][column] = '@';//encontra coordenada escolhida e coloca um navio nela
        }

        for(int i = 0; i < 5; i++){
            for(int j = 0; j < 5; j++){
                System.out.print(user.board[i][j] + " ");//imprime tabuleiro inicial
            }
            System.out.println();  
        }
    }
}
